KsApi/models/graphql/adapters/Backing+BackingFragment.swift

```diff
@@ -56,7 +56,18 @@ private func backingStatus(from backingFragment: GraphAPI.BackingFragment) -> Ba
 }
 
 private func backingReward(from backingFragment: GraphAPI.BackingFragment) -> Reward? {
-  guard let reward = backingFragment.reward?.fragments.rewardFragment else { return .noReward }
+  guard let reward = backingFragment.reward?.fragments.rewardFragment else {
+    let projectMinimumPledgeAmount: Int = backingFragment.project?.fragments.projectFragment.minPledge ?? 1
```

Comment:
Pretty interesting that we have to do this ourselves in GraphQL but didn't have to with V1? I wonder what happened there and why this bug is only happening on iOS (and not on Android)?

Reply:
well v1/ returned the no reward reward right? in graph the rewards we just the ones the user created, so we manually have to create the no reward reward in instances (like this one) where its' displayed. Just graph things lol

Reply:
I guess to elaborate Android did this accurately with GQL the first time on these pages.